docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -29,6 +29,18 @@ and then read metadata from each data file.
 Since Iceberg stores the paths to data files in the metadata files, it
 only consults the underlying file system for files that must be read.
 
+Requirements
+------------
+
+To use Iceberg, you need:
```

Comment:
At least one other connector requirements section (Accumulo connector) phrases this as "To connect to..." instead of "To use..."

Personally I prefer "To connect to" as long as that's technically accurate, but this should be made consistent.

Reply:
I'll bring this up when @mosabua and I start work on the Iceberg connector ticket.

Reply:
@mosabua said "to use" is correct because there is no running Iceberg system you can connect to. It's just a bunch of files.